my_enumerables.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    (0...size).each do |i|
```

Comment:
hay you can not use each in this project because you are building your own each method my_each and then use my_each where necessary in next methods.

Reply:
🤣🤣🤣 I got so caught into trying to understand yield, and self, and blocks, and proc, that I completely forgot about the requirements of the project.

Thank you for the feedback!